handlers/user.go

```diff
@@ -2455,7 +2455,7 @@ func NotificationWebhookPage(w http.ResponseWriter, r *http.Request) {
 
```

Comment:
The BIDS description says to remove the withdrawal checkbox from the /user/notifications view but it is still there. The description also says to disable the feature for e-mail notifications. Am I correct in assuming that you did not remove the possibility for e-mails to include withdrawals? Irregardless of whether the checkbox is available for the user to change.

Reply:
Right, I at first removed the checkbox but then had to revert is again, see the last comment in the BIDS.

You currently can't disable email notifications for specific notifications just for all in general.
And the validators set for those email notifications are then used for the webhooks.

So we decided to only add the webhooks option for now.